Snippets/Core/Core_6/Headers/Writers/HeaderWriterReply.cs

```diff
@@ -55,7 +52,7 @@ class MessageToReply :
         {
         }
 
-        class Mutator :
+        class Mutator : 
```

Comment:
Trailing whitespace! @andreasohlund Time to install https://marketplace.visualstudio.com/items?itemName=MadsKristensen.TrailingWhitespaceVisualizer!